StyleCop.Analyzers/StyleCop.Analyzers/Settings/ObjectModel/ReadabilitySettings.cs

```diff
@@ -9,11 +9,21 @@ namespace StyleCop.Analyzers.Settings.ObjectModel
     internal class ReadabilitySettings
     {
         /// <summary>
+        /// This is the backing field for the <see cref="lineLength"/> property.
+        /// </summary>
+        [JsonProperty("lineLength", DefaultValueHandling = DefaultValueHandling.Include)]
+        private int lineLength;
+
+        /// <summary>
         /// Initializes a new instance of the <see cref="ReadabilitySettings"/> class during JSON deserialization.
         /// </summary>
         [JsonConstructor]
         protected internal ReadabilitySettings()
         {
+            this.lineLength = 0;
         }
+
+        public int LineLength =>
+            this.lineLength;
     }
```

Comment:
LineLength should be named MaximumLineLength to provide more clarity to someone reading the `stylecop.json` file.